index_codec.md

```diff
@@ -3,46 +3,17 @@
 
 # Introduction
 
-Matroska aims to become THE standard of multimedia container formats. It was derived from a project called [MCF](http://mukoli.free.fr/mcf/mcf.html), but differentiates from it significantly because it is based on [EBML](https://github.com/Matroska-Org/ebml-specification/blob/master/specification.markdown) (Extensible Binary Meta Language), a binary derivative of XML. EBML enables significant advantages in terms of future format extensibility, without breaking file support in old parsers.
+Matroska aims to become THE standard of multimedia container formats. It stores interleaved and timestamped audio/video/subtitle data using various codec. To interpret the codec data, a mapping between the way the data are stored in Matroska and how they are understood by such codec is necessary.
```

Comment:
…and here we need singular: "…a mapping between the way the data are stored…" → "…is stored…".

Additionally: "…and how they are understood by such codec is necessary" This is difficult to understand. My guess is your "they" refers to the "the data", which are inherently singular in English. Meaning the whole sentence should probably be:

"To interpret the codec data, a mapping between the way the data is stored in Matroska and how it is understood by such a codec is necessary."